x-pack/plugins/infra/public/pages/metrics/hosts/index.tsx

```diff
@@ -29,6 +29,9 @@ const HOSTS_FEEDBACK_LINK =
 
 export const HostsPage = () => {
   const { isLoading, loadSourceFailureMessage, loadSource, source } = useSourceContext();
+  const {
+    services: { kibanaVersion, isCloudEnv, isServerlessEnv },
```

Comment:
Just realized that `kibanaVersion`,`isCloudEnv`,  and `isServerlessEnv` are under `services`. Should we move them out of the `services` object? They are just flags and not actual services

Reply:
Do we want to have a separate object [here](https://github.com/elastic/kibana/blob/main/src/plugins/kibana_react/public/context/context.tsx/#L17) in the context for those values? Should we call it kibanaEnvironment or kibanaSettings maybe? (those flags are about the kibana deployment/version so not sure what the best name is 🤔 wdyt?)

Reply:
As discussed I picked `kibanaEnvironment` and added the changes [here](https://github.com/elastic/kibana/pull/173758/commits/aaadb14d60cd9da76ea4a523148061201324a36e#diff-77464f5aecf6f5ea2dbf4e6e65b8edd91a35b6341963427639cc81a77ae27de0)